mbuild/compound.py

```diff
@@ -547,6 +547,8 @@ def mass(self, value):
     @property
     def charge(self):
```

Comment:
Add a docstring similar to the mass property while we have a chance.

Reply:
Good idea, I added some. Let me know if you think they need some changes.